src/pages/onboard/step4.page.tsx

```diff
@@ -0,0 +1,175 @@
+import { useEffect, useState } from 'react';
+import Lottie from 'react-lottie';
+import { useRouter } from 'next/router';
+import styled from '@emotion/styled';
+import { m, Variants } from 'framer-motion';
+import { useRecoilValue } from 'recoil';
+
+import Graphic from '@/components/graphic/Graphic';
+import Item from '@/components/item/Item';
+import { defaultFadeInUpVariants } from '@/constants/motions';
+import animationData from '@/constants/route-onboard/onboardingAnimation.json';
+import useUserOnboardMutaion from '@/hooks/api/onboard/useUserOnboardMutaion';
+import selectedOnboardCategory from '@/store/route-onboard/selectedOnboardCategory';
+import selectedOnboardItems from '@/store/route-onboard/selectedOnboardItems';
+import selectedOnboardTemplate from '@/store/route-onboard/selectedOnboardTemplate';
+import { WhiteBackgroundGlobalStyles } from '@/styles/GlobalStyles';
+
+const defaultOptions = {
+  loop: true,
+  autoplay: true,
+  animationData,
+  rendererSettings: {
+    preserveAspectRatio: 'xMidYMid slice',
+  },
+};
+
+const Step4 = () => {
+  const router = useRouter();
+
+  const selectedCategory = useRecoilValue(selectedOnboardCategory);
+  const selectedItems = useRecoilValue(selectedOnboardItems);
+  const selectedTemplate = useRecoilValue(selectedOnboardTemplate);
+
+  const [temp, setTemp] = useState<boolean>(false);
+  const { createUserMutation } = useUserOnboardMutaion();
+
+  useEffect(() => {
+    createUserMutation.mutate({
+      category: selectedCategory.type,
+      templateName: selectedTemplate ? selectedTemplate.title : '기본 소지품',
+      items: selectedTemplate
+        ? selectedTemplate.option.map((item) => item.name)
+        : selectedItems.map((item) => item.name),
+    });
+    const timer = setTimeout(() => setTemp(true), 3000);
+
+    return () => {
+      clearTimeout(timer);
+    };
+  }, []);
+
+  useEffect(() => {
+    if (!temp) return;
+    const timer = setTimeout(() => router.push('/'), 1500);
```

Comment:
```suggestion
    const timer = setTimeout(() => router.replace('/'), 1500);
```

뒤로가기를 방지하고자 replace로 하면 어떨까요!

Reply:
혹은 (베포 이후에) 사용자가 터치하면 넘어가는 ...? 그런 식으로도 동작할 수 있을 거 같네요 ! (그냥 떠오른 생각입니다)